Makefile

```diff
@@ -69,7 +69,7 @@ endif
 # setup opencv
 ifeq ($(USE_OPENCV), 1)
 	CFLAGS += -DMXNET_USE_OPENCV=1 $(shell pkg-config --cflags opencv)
-	LDFLAGS += $(shell pkg-config --libs opencv)
+	LDFLAGS += -ljpeg -lpng -lz $(shell pkg-config --libs opencv)
```

Comment:
These are dependencies of the suggested opencv build.  If linking to static opencv, which is done for some situations, like the pip package, there will be undefined references because the shared object no longer "hides' the calls to those libraries.   I didn't see anywhere else where it pulled in these dependencies.  Is there harm in defining these?